dcpy/test/lifecycle/ingest/test_transform.py

```diff
@@ -1,103 +1,69 @@
-from datetime import date
 import geopandas as gpd
 import pandas as pd
 import pytest
-from typing import cast
 import yaml
+from pydantic import TypeAdapter, BaseModel
 
-from dcpy.models import file
-from dcpy.models.lifecycle.ingest import Template
-from dcpy.models.lifecycle.ingest import LocalFileSource, Config, FunctionCall
+from dcpy.models.file import Format
+from dcpy.models.lifecycle.ingest import FunctionCall
 from dcpy.lifecycle.ingest import (
-    configure,
     transform,
     PARQUET_PATH,
 )
 
 from dcpy.utils import data
-from . import RESOURCES, TEST_DATA_DIR, FAKE_VERSION
+from . import RESOURCES, TEST_DATA_DIR
+
+
+class TestConfig(BaseModel):
+    """
+    Test pydentic class used to validate input yaml file
+    """
+
+    file_name: str
+    to_parquet_config: Format
```

Comment:
(just an idea, not a request to change asap)

I wonder if better names for these might be `file_config: FileConfig`? since it seems like the models included in `models.file.Format` are used as configs, and one of their attributes is named `format`

Reply:
I would prefer not calling something else a config. I think "File Format" is pretty intuitive (and a commonly used term). But maybe call the attribute of these different formats `type` rather than `format`